src/pages/wallet/index.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+import Navbar from "../../components/Navbar";
+import Footer from "../../components/Footer";
+import { Input } from "antd";
+import Button from "../../components/commons/Button";
+import { WarningOutlined } from "@ant-design/icons";
+
+function Wallet() {
+  return (
+    <>
+      <div className="wallet-container flex flex-col min-h-screen">
+        <Navbar />
+        <div className="flex flex-col justify-start items-center gap-5 flex-grow py-3">
+          <div className="flex justify-evenly w-full gap-5 py-5 px-10">
+            <div className="flex flex-col items-center justify-center ">
+              <p className="wallet-balance-title text-sm sm:text-2xl">
+                Your balance
+              </p>
+              <p className="wallet-balance-amount text-sm sm:text-2xl font-bold">
+                0 Lumens (XML)
```

Comment:
Where are we gonna show the user their balance after funding them?

Reply:
Solved in commit 7c074238754049be79477222701d97713ceb761b.